client/src/common/components/Comments/Comment.scss

```diff
@@ -8,6 +8,9 @@
 
   &__body {
     text-align: left;
+    width: calc(100% - 56px);
```

Comment:
you don't need this.
Just remove from .comment class `Justify-content `property